### PR TITLE
US-1889327: Suggestion/warning fixes for WPF.

### PR DESCRIPTION
**Pull request by poojapoojari**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by poojapoojari** [comment]
Hello @adegeo!

The 5th PR with 20 file edits is ready for technical review. Could you please review this PR?
Thank you!

**Comment by adegeo** [comment]
Hi @poojapoojari are you also supposed to add descriptions?

**Comment by poojapoojari** [comment]
Hi @adegeo!

I had updated the ms.date field as these 20 files had H2 heading updates (not just metadata changes). Also, these files were updated as per the ask in the [user story](https://dev.azure.com/mseng/TechnicalContent/_queries/edit/1889327/). However, as per your suggestion, I have updated the dates and added the description tag too. Could you please have a look and approve the same? :)

File: dotnet-desktop-guide/framework/wpf/advanced/localization-attributes-and-comments.md [hunk 1]
```diff
@@ -10,7 +12,7 @@ ms.assetid: ead2d9ac-b709-4ec1-a924-39927a29d02f
 [!INCLUDE[TLA#tla_winclient](../../../includes/tlasharptla-winclient-md.md)] localization comments are properties, inside XAML source code, supplied by developers to provide rules and hints for localization. [!INCLUDE[TLA#tla_winclient](../../../includes/tlasharptla-winclient-md.md)] localization comments contain two sets of information: localizability attributes and free-form localization comments. Localizability attributes are used by the [!INCLUDE[TLA2#tla_winclient](../../../includes/tla2sharptla-winclient-md.md)] Localization API to indicate which resources are to be localized. Free-form comments are any information that the application author wants to include.  
 
 <a name="Localizer_Comments_"></a>
-## Localization Comments  
+## How to: Add Localization Comments
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/localization-attributes-and-comments.md:15]:
```suggestion
## Add Localization Comments
```

**Comment by adegeo** [comment]
@poojapoojari thanks for these changes. I believe the description metadata is a suggestion and not a warning, but it should be added.

**Review by adegeo — APPROVED** [review]

**Pull request merged**